src/main/java/pageobject/HomePage.java

```diff
@@ -0,0 +1,46 @@
+package pageobject;
+
+import core.IWebDriverProvider;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import static core.Property.BASE_URL;
+
+public class HomePage extends AbstractPage {
+    @FindBy(id = "nav-link-accountList")
+    WebElement singIn;
+    @FindBy(xpath = "//div[@id='nav-logo']")
+    WebElement logo;
+    @FindBy(xpath = "//div[@role = 'main']")
+    WebElement homePage;
```

Comment:
bad naming, if that's the whole page, at least state it like homePageBody. Otherwise it's not clear what locator represents